app/src/main/java/ch/heigvd/pro/b04/android/Poll/PollActivity.java

```diff
@@ -51,6 +54,14 @@ protected void onCreate(Bundle savedInstanceState) {
 
         questionList.setAdapter(questionAdapter);
         questionList.setLayoutManager(manager);
+
+        state.getQuestionToView().observe(this, question -> {
+            Question q = question;
+
+            Intent intent = new Intent(this, QuestionActivity.class);
+            intent.putExtra("QUESTION_ID", q.getId());
+            startActivity(intent);
+        });
```

Comment:
Instead of creating a `q`, you can write the code as follows:
```
state.getQuestionToView().observe(this, question -> {
            Intent intent = new Intent(this, QuestionActivity.class);
            intent.putExtra("QUESTION_ID", question.getId());
            startActivity(intent);
        });
```